development/contributing.md

```diff
@@ -1,22 +1,44 @@
 ## Contributing
 
-### Git Commit Messages
+### Coding Best Practices
+
+#### Linting
+
+All code repositories adopt our standard linting rules found in the [flowforge/.github repository](https://github.com/flowforge/.github/blob/main/.eslintrc).
+
+We use [StandardJS](https://standardjs.com/), with one exception - 4 spaces not 2.
+
+If you're using VSCode, then we recommend using the [ESLint extenstion](https://github.com/Microsoft/vscode-eslint) and setting `all` for the `Eslint › Code Actions On Save: Mode` setting:
+
+<img width="429" alt="Screenshot 2022-04-07 at 10 11 04" src="https://user-images.githubusercontent.com/99246719/162164540-53cdf921-83a5-4d6e-9062-1552db24f453.png">
+
+In the case of working with `vue` or `njk` files (found in the [frontend](https://github.com/flowforge/flowforge/tree/main/frontend) and [website](https://github.com/flowforge/website) repositories), then you can add `vue` and `njk` to the `Eslint: Probe` setting in order to enable auto-formatting on save for these file types.
+
+<img width="478" alt="Screenshot 2022-04-07 at 10 09 15" src="https://user-images.githubusercontent.com/99246719/162164162-6822e32c-680d-4c5c-af73-64aae24b03a0.png">
+
+### Git Best Practices
+
+### Committing
+
+Take care when adding files to a commit. It's easy just to `git add -A` but this can result in commits and PRs being clogged with excessive changes that aren't linked to the actual issue/feature at hand.
```

Comment:
Propose an alternative: `git add -p`.

Reply:
I've made a change to be more explicit here, but have not included the `git add -p` as I've never used it myself, so wouldn't feel best placed to recommend it. Open to iteration with a future PR on my changes though